webapp/channels/src/components/post_view/post_time/post_time.tsx

```diff
@@ -97,8 +93,8 @@ export default class PostTime extends React.PureComponent<Props> {
                 title={
                     <Timestamp
                         value={eventTime}
-                        ranges={POST_TOOLTIP_RANGES}
                         useSemanticOutput={false}
+                        useDate={{weekday: 'long', day: 'numeric', month: 'long', year: 'numeric'}}
```

Comment:
Since we are always passing the same value, can we move it to a constant (similar to how POST_TOOLTIP_RANGES was) so we don't pass a new reference on each render?